app/policies/user_policy.rb

```diff
@@ -1,2 +1,14 @@
 class UserPolicy < ApplicationPolicy
+  def index?
+    user && user.facility_access?
```

Comment:
I think only `superuser` can see other users on the platform.
```suggestion
    user && user.superuser?
```